src/app/components/FrostedGlassPromo/index.test.tsx

```diff
@@ -17,9 +18,18 @@ import {
 } from './fixtures';
 
 import Promo from '.';
-
-/* eslint-disable react/prop-types */
-const Component = ({ service = 'mundo', variant, ...rest }) => {
+import { PromoProps } from './types';
+
+interface Props extends PromoProps {
+  service?: Services;
+  variant?: Variants;
+}
+
+const Component = ({
+  service = 'mundo',
+  variant,
+  ...rest
```

Comment:
Is it also worth adding/amending the component standards to talk about when we still allow props spreading? This was an issue when migrating the `VisuallyHiddenText` component. The remaining `html` attributes needed to be spread to allow the tests to pass. Perhaps the usage of it here is slightly different but thought i'd mention it

Reply:
Yea possibly. For tests I think its somewhat ok to do it since the fixture data can vary so much on what is being passed into the component.

The `htmlAttributes` one needed to be spread, since it wouldn't be practical to list every permutation of HTML attributes that could be passed to the component.